Sources/HTMLKit/Internal/Features/Conversion/Converter.swift

```diff
@@ -104,6 +104,22 @@ public class Converter {
         }
     }
     
+    /// Converts an HTML component to be put into an existing Page or View layout.
+    /// The whole html string needs to be inside a tag.
+    /// For example, multiple `div`s will give an error. They need to be inside another `div`.
+    public func convert(html: String) throws -> String {
+        // The converter puts a new line inside a single quote Swift string, which breaks the code. To avoid that:
```

Comment:
If the code is not valid, the converter sees the content as a single text node. Do you mean it by that?